packages/cdktf-cli/bin/cdktf

```diff
@@ -1,2 +1,2 @@
-#!/usr/bin/env node
-require('./cdktf.js');
\ No newline at end of file
+#!/usr/bin/env node --experimental-abortcontroller
```

Comment:
Maybe we can remove this on node v14.17?

Reply:
https://nodejs.org/docs/latest-v14.x/api/globals.html#globals_class_abortcontroller seems to indicate that it's needed for 14.X